pybossa/view/admin.py

```diff
@@ -188,7 +188,7 @@ def export_users():
         return redirect(url_for('.index'))
     if fmt not in export_formats:
         abort(415)
-    mail_queue.enqueue(export_all_users, fmt, current_user.email_addr)
+    mail_queue.enqueue(export_all_users, fmt, current_user.email_addr, **dict(job_timeout='1200s'))
```

Comment:
you can also do
```python
mail_queue.enqueue(export_all_users, fmt, current_user.email_addr, job_timeout='1200s')
```

Reply:
In this way, timeout could become arguments of export_all_users according to this https://github.com/rq/rq/blob/5328a6252b7fec6b6df5875458750ba8629fec86/rq/queue.py#L297